docs/Reference.md

```diff
@@ -85,6 +85,37 @@ It is useful to note that snowflake IDs are just numbers with a timestamp, so wh
 timestamp_ms - DISCORD_EPOCH << 22
 ```
 
+## ID Serialization
+
+There are some cases in which our API and Gateway may return IDs in an unexpected format. Internally, Discord stores IDs as integer snowflakes. When we seralize IDs to JSON, we transforms `bigints` into strings. Given that all Discord IDs are snowflakes, you should always expect a string.
+
+However, there are cases in which your passing something to our API will instead return you IDs serialized as an integer; this is the case when you send our API or Gateway a value in an `id` field that is not `bigint` size. For example, when requesting `GUILD_MEMBERS_CHUNK` from our gateway:
+
+```
+// Send
+{
+  op: 8,
+  d: {
+    guild_id: '308994132968210433',
+    user_ids: [ '123123' ]
```

Comment:
This isn't documented in the request guild members docs, should it be added there?

Reply:
Seems fine I guess? I can always remove it if needed :blobshrug: